src/quant/test_models.py

```diff
@@ -0,0 +1,24 @@
+"""Apply unit test to models"""
+from . import models as sc
+
+
+def test_model():
```

Comment:
this should be in the `tests` directory at the top of the project hierarchy, that way it'll get run with `pytest`.

Reply:
probably best to name the file `test_quant.py`